Sources/NIOHTTP2/ConnectionStateMachine/ConnectionStreamsState.swift

```diff
@@ -125,7 +112,6 @@ struct ConnectionStreamState {
     ///     - modifier: A block that will be invoked to modify the stream state, if present.
     /// - throws: Any errors thrown from the creator.
     /// - returns: The result of the state modification, as well as any state change that occurred to the stream.
-    @inline(__always)
```

Comment:
We have to remove this because otherwise the 5.0 compiler crashes. :shrug: We don't much care about perf on 5.0 anymore, it's not the best target, so we'll accept this change. Newer compilers didn't need this anyway.